Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml

```diff
@@ -779,12 +686,26 @@ script:
     - description: The value of the attribute to change.
       name: attributevalue
       required: true
-    - description: Root. For example, DC=domain,DC=com. By default, the Base DN configured
-        for the instance is used.
+    - description: Root. For example, DC=domain,DC=com. By default, the Base DN configured for the instance is used.
       name: basedn
     description: Updates attributes of an existing Active Directory group.
     name: ad-update-group
-  dockerimage: demisto/py3-tools:0.0.1.30065
+  - name: ad-modify-password-never-expire
+    arguments:
```

Comment:
Please add the sAMAccountName argument (if it is necessary).
If it doesn't - remove it from the command itself.